lib/IRGen/IRGenSIL.cpp

```diff
@@ -2041,15 +2084,15 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
 
     switch (attrInfo.second) {
     case GraphOperationInfo::ArgumentLowering::Input:
-      llvm_unreachable("Input classes cannot exist for attributes");
+      assert(0 && "Input classes cannot exist for attributes");
```

Comment:
What's the reason not to use unreachable?

Reply:
Assertion gives us a nice crash stack trace pointing to this source location. llvm_unreachable() does not. Once we complete the impl, using llvm_unreachable should be fine on invariant checks. But in the mean time, using assertion makes it easier to debug test cases.